dev-server/ui-playground/index.js

```diff
@@ -1,10 +1,24 @@
 import { registerIcons } from '@hypothesis/frontend-shared';
 import { render } from 'preact';
 
-import PlaygroundApp from './components/PlaygroundApp';
+import ButtonPatterns from './components/ButtonPatterns';
+
+import PlaygroundApp from './shared/components/PlaygroundApp';
 
 import sidebarIcons from '../../src/sidebar/icons';
 registerIcons(sidebarIcons);
 
 const container = document.querySelector('#app');
-render(<PlaygroundApp />, /** @type Element */ (container));
+
+const extraRoutes = [
+  {
+    route: '/buttons',
+    title: 'Buttons',
+    component: ButtonPatterns,
```

Comment:
The heading at the top of the page and document title for both the "Common Patterns > Buttons" and "Application Patterns > Buttons" is currently the same. Also I think it would be nice if the route path more closely matched the visual structure of the sidebar.

Reply:
>  Also I think it would be nice if the route path more closely matched the visual structure of the sidebar.

Sorry, could you clarify?